Source/CesiumRuntime/Private/Tests/TestGeoTransforms.cpp

```diff
@@ -0,0 +1,127 @@
+#include "CesiumGeospatial/Ellipsoid.h"
+#include "CesiumUtility/Math.h"
+#include "GeoTransforms.h"
+#include "GeoTransformsNew.h"
+#include "Misc/AutomationTest.h"
+
+using namespace CesiumGeospatial;
+using namespace CesiumUtility;
+
+BEGIN_DEFINE_SPEC(
+    FTestGeoTransforms,
+    "Cesium.GeoTransforms",
+    EAutomationTestFlags::ApplicationContextMask |
+        EAutomationTestFlags::ProductFilter)
+END_DEFINE_SPEC(FTestGeoTransforms)
+
+void FTestGeoTransforms::Define() {
+  Describe("TransformLongitudeLatitudeHeightToUnreal", [this]() {
```

Comment:
Might be nice to add more unit tests for `GeoTransformsNew`? Doesn't have to hold up this PR, it's already a step up from none 😛